### ISSUE TITLE
Console scroll bar is not working correctly.

### ISSUE THREAD

**Issue by RicardRomani** [a85/POSTMan-Chrome-Extension#10246]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

![Animation](https://user-images.githubusercontent.com/52193854/130584243-7965dd00-f56c-49ac-9c0f-6709e1f84165.gif)

When I try to scroll down or up in the console it doesn't works as expected. Other scroll bars work perfectly.

### Steps To Reproduce

1. I make a request
2. I open the console.
3. I expand the request info -> Make scroll bar visible
4. Scroll in the console log

### Environment Information

```markdown
- Operating System: Windows 10 Pro
- Platform Type: Postman for Windows
- Postman Version: Version 8.11.0
```

**Comment by UmedJadhav** [comment]
@RicardRomani , we have added a fix for this which should be released by today / tomorrow via v8.11 release
cc: @coditva

**Comment by RicardRomani** [comment]
@UmedJadhav after updating to **v8.11.1** console is working fine now! Thank you team! 👍
